.github/workflows/release-stable.yml

```diff
@@ -9,9 +9,21 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Check ref_type
```

Comment:
maybe we should also add this "guarding" first step to the release snapshot workflow?